src/main/java/frc/team4909/Drivetrain.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Team 4909, Bionics
+ * Billerica Memorial High School
+ *
+ * Copyright:
+ *   2021 Bionics
+ *
+ * License:
+ *   MIT: https://opensource.org/licenses/MIT
+ *   See the LICENSE file in the project's top-level directory for details.
+ */
+
+package frc.team4909;
+
+import edu.wpi.first.wpilibj.SerialPort;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import edu.wpi.first.wpilibj.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.wpilibj.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc.bionic.swerve.IDrivetrainSubsystem;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.kauailabs.navx.frc.*;
+
+public class Drivetrain implements Subsystem, IDrivetrainSubsystem {
+  public team4909SwerveModule swerveRF; // right front
+  public team4909SwerveModule swerveLF; // left front
+  public team4909SwerveModule swerveLR; // left rear
+  public team4909SwerveModule swerveRR; // right rear
```

Comment:
Are these definitions and the instantiations of them below the only changes you had to make to Drivetrain between peyton and team4909? If it's mostly common, we should either create an abstract class that each extends with its own little details, or move the class to a bionic.Drivetrain and add parameters for the few things that differ. Let's diff the two files and see if there are any other changes, unless you recall any.